examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Maybe it was explained in another notebook but we could specify that <code>merlin.io.Dataset</code> is expecting the .pb schema file within the <code>train</code> folder, and this file was automatically saved by NVTabular workflow?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>